apps/cow-tools/src/app/milkman/page.tsx

```diff
@@ -1,6 +1,20 @@
+"use client";
+
+import { NetworkChainId, NetworkFromNetworkChainId } from "@bleu-fi/utils";
+import { useSafeAppsSDK } from "@gnosis.pm/safe-apps-react-sdk";
 import { redirect } from "next/navigation";
+import { gnosis, goerli } from "viem/chains";
 
 export default function Page() {
-  //TODO: change redirect to default network BLEU-366
-  redirect("/milkman/goerli");
+  const {
+    safe: { chainId },
+  } = useSafeAppsSDK();
+
+  if (chainId == goerli.id || chainId == gnosis.id) {
```

Comment:
later on you can create a function to check if the chain ID is supported, we have something like this on balancer-tools